DigitalExcellence/dex-backend
# Feature/107 role based authorization

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Log in as privileged user.
2. request privilged api endpoints.
3. change privileges to remove the used scopes.
4. set role to testing user.
5. request the privilged api endpoint.
6. should be forbidden.

## Link to issue

Closes: #107
